tbf-challenge/teamwork-be
# Add CI to the project

#### Pre-Submission Checklist






#### Checklist:





#### What does this PR do?
This  pull request setup CI workflows for the project

#### How should this be manually tested?
Each time a code is pushed or a pull request is created the workflow should build and test them


#### What are the relevant issues?
https://github.com/tbf-challenge/teamwork-be/issues/9
